docs/src/main/asciidoc/spring-cloud-netflix.adoc

```diff
@@ -433,6 +433,10 @@ dependencyManagement {
 ----
 ====
 
+=== `defaultOpenForTrafficCount` and its effect on EurekaServer warmup time
+
+Netflix Eureka's `waitTimeInMsWhenSyncEmpty` setting is not taken into account in Spring Cloud Eureka server at the beginning.In order to enable the warmup time, set `eureka.server.defaultOpenForTrafficCount=0`.It will enable the warmup time.
```

Comment:
Actually, we can remove the last sentence as well, since this information is already covered in the second sentence.